javascript.js

```diff
@@ -0,0 +1,43 @@
+const products = [
+  {
+    imgSrc: 'images/imac.jpeg',
+    altText: 'Apple iMac',
+    title: 'APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)',
+    code: '195434',
+    reviews: 5,
+    price: '$2,199',
+  },
+  // Другие карточки
+];
+
+const productContainer = document.getElementById('product-container');
+
+products.forEach((product) => {
+  const card = document.createElement('div');
+  card.className = 'card';
+  card.setAttribute('data-qa', 'card');
+
+  card.innerHTML = `
+    <img class="card__img" src="${product.imgSrc}" alt="${product.altText}" />
```

Comment:
Consider adding error handling or a default image in case `product.imgSrc` is not a valid URL or the image fails to load.